### PR TITLE
Add font stretch CSS example

### LINKED ISSUE #573: CSS examples: need example for font-stretch
https://developer.mozilla.org/en-US/docs/Web/CSS/font-stretch

Syntactically this looks quite simple. The only tricky bit is finding a font that supports the stretched and condensed variants.

For consistency it would be good to base the example on one of the existing font examples:
https://developer.mozilla.org/en-US/docs/Web/CSS/font-family
https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight
etc.

### PR DESCRIPTION
**Pull request by tannerdolby**
Fixes #573

I used the "Regular" style `.ttf` file from [League Mono](https://www.theleagueofmoveabletype.com/league-mono), but the text seems a bit "light", might just be me. Below is a demo of the working example in Chrome, (works in Firefox too).

https://user-images.githubusercontent.com/48612525/114841578-dafe1900-9d8c-11eb-994e-bba2cad087ce.mov

### CODE REVIEW

**Comment by escattone** [comment]
@tannerdolby Just FYI, if you merge the `master` branch into your PR, the build should work again.

**Comment by tannerdolby** [comment]
@escattone Ok thanks for the heads up. Want me to merge `master` into this PR with another commit or is it fine in its current form?

**Comment by escattone** [comment]
> @escattone Ok thanks for the heads up. Want me to merge `master` into this PR with another commit or is it fine in its current form?

@tannerdolby You could either merge `master` into this PR with another commit, or rebase your PR on `master` if you'd prefer. Thanks.

**Comment by tannerdolby** [comment]
@escattone I merged my feature branch with the up-to-date `master` and the build is working now 👍

**Comment by tannerdolby** [comment]
@escattone Let me know if you need me to make any changes before this is ready to merge.

**Comment by escattone** [comment]
@wbamberg I just added you to this repo with maintainer privileges (which I should have done long ago, sorry!), and then also added you as the reviewer.

@tannerdolby All you need is @wbamberg's review/approval. Thanks for your work on this!

**Comment by tannerdolby** [comment]
@escattone No problem! And thank you for initiating the review/approval from @wbamberg

File: live-examples/css-examples/fonts/font-stretch.css
Comment (wbamberg) [inline live-examples/css-examples/fonts/font-stretch.css:5]:
```suggestion
    font-weight: 400;
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks @escattone ! It's been a while since I reviewed one of these :).

Thanks for the PR @tannerdolby ! I agree it does look a bit light. I suggested changing `font-weight`, unless there's a reason you went with `100` here. Apart from that this looks great.

File: live-examples/css-examples/fonts/font-stretch.css
Comment (tannerdolby) [reply live-examples/css-examples/fonts/font-stretch.css:5]: No problem @wbamberg ! Ah yes, I must have have forgotten about changing the `font-weight` back haha, good catch. Changing it to `font-weight: 400` looks much better 👍

**Review by wbamberg — APPROVED** [review]
👍 awesome, thanks @tannerdolby !

**Comment by wbamberg** [comment]
Thanks for your contribution @tannerdolby ! It's already live at https://interactive-examples.mdn.mozilla.net/pages/css/font-stretch.html .

If you want to do the last bit, so this example shows up on MDN, we need to update https://developer.mozilla.org/en-US/docs/Web/CSS/font-stretch over in https://github.com/mdn/content/tree/main/files/en-us/web/css/font-stretch , replacing the big grey box at the top with a macro call like:

```
<div>{{EmbedInteractiveExample("pages/css/font-stretch.html")}}</div>
```

Then we can merge that and the new example will be live on MDN.

**Comment by tannerdolby** [comment]
Fully completed when [#4608](https://github.com/mdn/content/pull/4608) is merged in `mdn/content` 🎉

**Pull request merged**
